mlflow/keras.py

```diff
@@ -118,8 +129,7 @@ def log_model(keras_model, artifact_path, conda_env=None, **kwargs):
                       contained in :func:`get_default_conda_env()`. If ``None``, the default
                       :func:`mlflow.keras.get_default_conda_env()` environment is added to
                       the model. The following is an *example* dictionary representation of a
-                      Conda environment::
-
+                      Conda environment:
```

Comment:
Can we revert this change? The double colon signifies the beginning of a literal block in RST (http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#literal-blocks).